tests/post/steps/conftest.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from configparser import ConfigParser
+from itertools import chain
+
+import pytest
+
+
+@pytest.fixture(scope="session")
+def version():
+    """Fixture that read metalk8s version.
+
+    Version is extracted from _build/root/product.txt
+    """
+    version_file = "_build/root/product.txt"
+    parser = ConfigParser()
+
+    with open(version_file) as stream:
+        # add a section header for configparser
+        stream = chain(("[top]",), stream)
+        parser.readfp(stream)
+
+    return {k: v for k, v in parser.items('top')}
+
+
+@pytest.fixture
+def verify_kubeapi_service(host):
```

Comment:
we can put common steps in the conftest.py (that is why I created it in the first place)

Reply:
In fact I is not really common steps as it is used by a `Given` steps in the first test and a `Then` step in the 2nd test. I did not find a way to alias the 2 steps, that's why I create the common fixture in the `conftest.py`

Reply:
I moved it to the conftest